vaadin-grid-flow-integration-tests/src/test/java/com/vaadin/flow/component/grid/it/GridMultiSelectionColumnPageIT.java

```diff
@@ -76,6 +103,11 @@ public void noSelectOnRowItemClick() {
                 .get().click();
         Assert.assertEquals("No selection event should be fired", "",
                 findElement(By.id("selected-item-count")).getText());
+
+        // Test switch selection mode from Multi to single mode before adding the grid to DOM
+        // By checking the vaadin-grid-cell-content
+        WebElement gridSelectionMode = findElement(By.id("in-testing-multi-selection-mode-grid-single"));
+        Assert.assertEquals(gridSelectionMode.findElements(By.tagName("vaadin-checkbox")).size(), 0);
```

Comment:
This is the method definition of `Assert.assertEquals`
```java
static public void assertEquals(long expected, long actual)
```
So the expected result should be the first parameter, and the actual result the second.

Reply:
done